specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/examples/communityGallery/GetACommunityGallery.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "location": "myLocation",
+    "api-version": "2021-07-01",
+    "publicGalleryName": "publicGalleryName"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "location": "myLocation",
+        "name": "publicGalleryName",
+        "type": "Microsoft.Compute/CommunityGallery",
```

Comment:
>"Microsoft.Compute/CommunityGallery" [](http://example.com/codeflow?start=16&length=36)

Type should be 'Microsoft.Compute/Locations/CommunityGallery"

[https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderRegistrationRoutingTypes.html?q=routing#locationbased](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderRegistrationRoutingTypes.html?q=routing#locationbased)

Reply:
updated. thanks!